src/Components/SoundWave/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useRef, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+
```

Comment:
I tried testing out this component and I couldn't because I was having a hard time getting the mp3 file. Looks like we need to do some extra configuring of webpack to get it to work. Since this component is a little complex I am going to ask that a story is created for this component so it will be easier for to determine if it is working or not.

- [ ] Create story for this component. See readme and view DrumPad's story for more information and an example.